picard/file_dilaog.py

```diff
@@ -0,0 +1,274 @@
+from PyQt5.QtGui import *
+from PyQt5.QtWidgets import *
+from PyQt5.QtCore import *
```

Comment:
Never make start imports. Its better to be verbose about where we are getting our classes and variables from.